packaging/windows-exe/rabbitmq_nsi.in

```diff
@@ -132,6 +132,7 @@ Section "RabbitMQ Service" RabbitService
   ReadEnvStr $1 "HOMEDRIVE"
   ReadEnvStr $2 "HOMEPATH"
   CopyFiles "$WINDIR\.erlang.cookie" "$1$2\.erlang.cookie"
+  CopyFiles "$SystemRoot\System32\config\systemprofile\.erlang.cookie" "$1$2\.erlang.cookie"
```

Comment:
When the source file is missing, `CopyFiles` logs an error. That's how I found this issue.

Should we use `IfFileExists` before calling `CopyFiles` to avoid this error message?

Reply:
That would make sense to me.